build/commands/lib/calculateFileChecksum.test.js

```diff
@@ -7,39 +7,42 @@ const dirPrefixTmp = 'brave-browser-test-calculate-file-checksum-'
 const testFile1Name = 'file1'
 const testFile1InitialContent = 'this is a test'
 const encoding = 'utf8'
-let testDirPath, testFile1Path
 
-beforeEach(async function () {
-  // Test directory
-  testDirPath = await fs.mkdtemp(path.join(os.tmpdir(), dirPrefixTmp))
-  // Initial test file
-  testFile1Path = path.join(testDirPath, testFile1Name)
-  await fs.writeFile(testFile1Path, testFile1InitialContent, encoding)
-})
+describe('Checksums', function() {
```

Comment:
Since this is a single-module test, lets call this `calculateFileChecksum`?

Reply:
Sure. Done.